test/integration/addresses.js

```diff
@@ -97,43 +90,31 @@ describe('bitcoinjs-lib (addresses)', function () {
     assert.strictEqual(address, '3P4mrxQfmExfhxqjLnR2Ah4WES5EB1KBrN')
   })
 
-  it('can support the retrieval of transactions for an address (via 3PBP)', function (done) {
-    const keyPair = bitcoin.ECPair.makeRandom()
-    const { address } = bitcoin.payments.p2pkh({ pubkey: keyPair.publicKey })
-
-    dhttp({
-      method: 'GET',
-      url: 'https://blockchain.info/rawaddr/' + address
-    }, function (err, result) {
-      if (err) return done(err)
-
-      // random private keys [probably!] have no transactions
-      assert.strictEqual(result.n_tx, 0)
-      assert.strictEqual(result.total_received, 0)
-      assert.strictEqual(result.total_sent, 0)
-      done()
-    })
-  })
-
-  // other networks
+  // examples using other network information
   it('can generate a Testnet address', function () {
-    const testnet = bitcoin.networks.testnet
-    // in production: const keyPair = bitcoin.ECPair.makeRandom({ network: testnet })
-    const keyPair = bitcoin.ECPair.makeRandom({ network: testnet, rng: rng })
-    const wif = keyPair.toWIF()
-    const { address } = bitcoin.payments.p2pkh({ pubkey: keyPair.publicKey, network: testnet })
-
-    assert.strictEqual(address, 'mubSzQNtZfDj1YdNP6pNDuZy6zs6GDn61L')
-    assert.strictEqual(wif, 'cRgnQe9MUu1JznntrLaoQpB476M8PURvXVQB5R2eqms5tXnzNsrr')
+    const keyPair = bitcoin.ECPair.makeRandom({ network: TESTNET })
+    const { address } = bitcoin.payments.p2pkh({ pubkey: keyPair.publicKey, network: TESTNET })
+
+    // bitcoin testnet P2PKH addresses start with a 'm'
+    assert.strictEqual(address.startsWith('m'), true)
```

Comment:
This is 

```javascript
assert.strictEqual(address.startsWith('m') || address.startsWith('n'), true)
```

Since testnet p2pkh start with "m" or "n"